src/ngraph/runtime/gpu/cuda_error_check.hpp

```diff
@@ -16,17 +16,16 @@
 
 #pragma once
 
-#include <iostream>
-#include <sstream>
-#include <stdexcept>
-#include <stdint.h>
-#include <string>
-
```

Comment:
nit: Better to separate the std lib headers from the cuda-related headers?

Reply:
I was 50:50 on this.